jsk_pcl_ros/sample/sample_color_histogram_matcher.launch

```diff
@@ -0,0 +1,44 @@
+<launch>
```

Comment:
Can you add test?
Only testing if output topics are published is ok.

Reply:
I added test for a color_histogram_matcher node, I confirmed correctly pass a rostest on my environment.